js-envs/src/main/scala/org/scalajs/jsenv/nodejs/JSDOMNodeJSEnv.scala

```diff
@@ -88,7 +89,14 @@ class JSDOMNodeJSEnv(
            |})();
            |""".stripMargin
       }
-      Seq(new MemVirtualJSFile("codeWithJSDOMContext.js").withContent(jsDOMCode))
+      val codeFile = new File(jsdomDirectory, "codeWithJSDOMContext.js")
```

Comment:
> IMO the natural way of addressing this issue would be to give an extra list of paths where it can look for modules. Would assume there exists a command-line option for that which we can use.

If I understand correctly, this way is (unfortunately) deprecated: https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders